llvm/unittests/DebugInfo/PDB/PDBVariantTest.cpp

```diff
@@ -37,7 +37,7 @@ using TestTypes = testing::Types<bool, int8_t, uint8_t, int16_t, uint16_t,
 
 } // namespace
 
-TYPED_TEST_SUITE(PDBVariantIntegerTest, TestTypes);
+TYPED_TEST_SUITE(PDBVariantIntegerTest, TestTypes, );
```

Comment:
This looks funny to me, but it does seem to fix the warning and we have lots of code like this already (e.g. https://github.com/llvm/llvm-project/blob/6c773a87013cf82b68ba9be4672e3908a9ab6010/llvm/unittests/ADT/BitVectorTest.cpp#L24).